pages/common/firebase.md

```diff
@@ -1,25 +1,28 @@
 # firebase
 
-> Firebase Command Line Interface (CLI) Tools.
-> Used to test, manage, and deploy your Firebase project from the command line.
+> Test, manage, and deploy Firebase projects from the command line.
 > More information: <https://github.com/firebase/firebase-tools>.
 
-- Login:
+- Login to https://console.firebase.google.com:
```

Comment:
Should we maybe highlight URLs somehow? Maybe even making them a link with `<...>`?

Reply:
Not a bad idea, I think probably the angle brackets won't get removed in most clients, but that might not matter.  @sbrl?

Reply:
Shouldn't that end up the same as the `More information` link?

Reply:
Yes, but I was thinking that it wouldn't be as acceptable there.  I don't think it probably matters though.
```suggestion
- Login to <https://console.firebase.google.com>:
```